ignite/pkg/cliui/view/accountview/account_test.go

```diff
@@ -0,0 +1,54 @@
+package accountview
```

Comment:
```suggestion
package account_test
```

Reply:
fixed. note: changed it so accountview_test, since the original package name is accountview

Reply:
Sorry, I made a mistake. We can still call `accountview` or `accountview_test`. I thought the package was `account`, and the tests `accountview`